src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml

```diff
@@ -603,15 +603,17 @@
     <page-system-groupmanager>
         <name>System: Group manager</name>
         <patterns>
-            <pattern>system_groupmanager.php*</pattern>
+            <pattern>ui/auth/group</pattern>
+            <pattern>api/auth/group/*</pattern>
         </patterns>
     </page-system-groupmanager>
-    <page-system-groupmanager-addprivs>
-        <name>System: Group Manager: Add Privileges</name>
+    <page-system-groupmanager>
```

Comment:
`page-system-groupmanager` tag is repeated here.. I don't think that is your intention... maybe you wanted `page-system-privmanager`?